src/main/java/gov/cms/mat/cql_elm_translation/utils/HumanReadableUtil.java

```diff
@@ -58,14 +58,24 @@ public static List<String> getMeasureTypes(Measure measure) {
   }
 
   public static String getStratification(Measure measure) {
+    // Collects and returns all stratification descriptions for display
     if (CollectionUtils.isNotEmpty(measure.getGroups())) {
+      String allDescriptions = "";
       for (Group group : measure.getGroups()) {
+        if (allDescriptions.length() > 0
+            && CollectionUtils.isNotEmpty(group.getStratifications())) {
+          allDescriptions += "\n";
```

Comment:
This feels unnecessary since `Collectors.joining()` below is doing the same thing, but single space. If we want double space, then the joining argument could be changed.

Reply:
This adds a new line when it goes to a new, nonempty group. Otherwise it will go (2 in first group, one in second):
```
Strat1
Strat2Strat3
```

Reply:
But this might not matter anyway since the method of building the output is changing

Reply:
The newline can be moved to right after the `allDescriptions` append and then `trim()` the extra newline on the return.